.github/workflows/check-code-quality.yml

```diff
@@ -20,12 +20,11 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
       - name: Prepare flutter
-        uses: subosito/flutter-action@e938fdf56512cc96ef2f93601a5a40bde3801046 # v2.19.0
+        uses: subosito/flutter-action@v2 # v2.19.0
```

Comment:
I think the other approach is safer, using the fixed commit hash
`subosito/flutter-action@e938fdf56512cc96ef2f93601a5a40bde3801046 # v2.19.0`